### PR TITLE
Fix geth deploy

### PR DESCRIPTION
**Pull request by yann300**
fix https://github.com/ethereum/browser-solidity/issues/412

### CODE REVIEW

File: src/app/renderer.js [hunk 1]
```diff
@@ -165,8 +165,8 @@ Renderer.prototype.contracts = function (data, source) {
       code += 'var ' + inp.name + ' = /* var of type ' + inp.type + ' here */ ;\n'
     })
 
-    code += 'var ' + contractName + 'Contract = web3.eth.contract(' + jsonInterface.replace('\n', '') + ');' +
-      '\nvar ' + contractName + ' = ' + contractName + 'Contract.new('
+    code += 'var ' + contractName.replace(':', '_') + 'Contract = web3.eth.contract(' + jsonInterface.replace('\n', '') + ');' +
```
Comment (chriseth) [inline src/app/renderer.js]: I think you also have to replace `/`

Comment (axic) [inline src/app/renderer.js]: `:`, `.`, `/` are replaced in solc, probably should follow the same here.

File: src/app/renderer.js [hunk 2]
```diff
@@ -165,6 +165,7 @@ Renderer.prototype.contracts = function (data, source) {
       code += 'var ' + inp.name + ' = /* var of type ' + inp.type + ' here */ ;\n'
     })
 
+    contractName = contractName.replace(/(:|\.|\/|\s)/g, '_')
```
Comment (chriseth) [inline src/app/renderer.js]: I would propose to use`/[:./]/g`

Comment (yann300) [inline src/app/renderer.js]: ok

**Pull request merged**